src/main/java/nextstep/fp/StreamStudy.java

```diff
@@ -15,19 +15,21 @@ public static long countWords() throws IOException {
                 .get("src/main/resources/fp/war-and-peace.txt")), StandardCharsets.UTF_8);
         List<String> words = Arrays.asList(contents.split("[\\P{L}]+"));
 
-        long count = 0;
-        for (String w : words) {
-            if (w.length() > 12) count++;
-        }
-        return count;
+        return words.stream().filter(w -> w.length() > 12).count();
```

Comment:
적절한 개행을 추가해서 가독성을 높여보시면 좋을 것 같아요
```suggestion
        return words.stream()
                .filter(w -> w.length() > 12)
                .count();
```